src/synopsys-action/synopsys-bridge.ts

```diff
@@ -17,17 +17,28 @@ export class SynopsysBridge {
     if (!SYNOPSYS_BRIDGE_PATH) {
       info('Synopsys Bridge path not found in configuration')
       info('Looking for synopsys bridge in default path')
+      console.log(`This platform is ${process.platform}`);
```

Comment:
I believe we didn't format it and didn't check for linting. This could be leading to linting issue.